src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -825,6 +836,17 @@ class PersonAgent(
       }
   }
 
+  def replanningReasonBuilder(data: BasePersonData, prefix: String): String = {
+    val reasonBuilder = new StringBuilder(prefix)
```

Comment:
How about this for terseness (especially since `None` has no effect and it is minimal concatenation):

```
(data.map{
  case mode if mode.isRideHail => s"$prefix RideHail"
  case mode if mode.isMassTransit || mode.isTransit => s"$prefix PT"
}).getOrElse(prefix)
```

Reply:
this refracting is done